.circleci/config.yml

```diff
@@ -0,0 +1,216 @@
+version: 2.1
+
+######################################################################
+#
+# Start of general purpose config. These can plausibly go into orbs
+#
+######################################################################
+
+# Default settings for executors
+
+defaults: &defaults
+  working_directory: ~/repo
+
+# Runners for OpenJDK 8/11/16/17
+
+executors:
+  openjdk8:
+    docker:
+      - image: circleci/clojure:openjdk-8-lein-2.9.1-node
```

Comment:
I think there's a newer version of this image already. Btw, do we really need `node` here?